### PR TITLE
Added Node Red

### PR DESCRIPTION
**Pull request by gseva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ _Business Intelligence_
   * [Healthchecks](https://healthchecks.io/) - Django app which listens for pings and sends alerts when pings are late. ([Source Code](https://github.com/healthchecks/healthchecks)) `BSD` `Python`
   * [Huginn](https://github.com/cantino/huginn) - Allows you to build agents that monitor and act on your behalf - `MIT` `Ruby`
   * [Http2pic](https://http2pic.haschek.at/) - Website screenshots/renderer. It uses the wkhtmltox to render websites with various options. ([Source Code](https://github.com/chrisiaut/http2pic)) `Apache 2.0` `PHP/Javascript`
+  * [Node RED](http://nodered.org/) - A browser-based flow editor that helps you wiring hardware devices, APIs and online services to create IoT solutions. ([Source Code](https://github.com/node-red/node-red)) `Apache 2.0` `Node.js`
```
Comment (nodiscc) [inline README.md]: `Nodejs`

**Comment by nodiscc** [comment]
Thank you, good to merge.

@Kickball @AndyR207 I'm starting to think we should create an IoT subsection for Automation. Automation was originally downloaders/automatic archivers/rss generators etc. Node Red is interesting but it's a tool to create hardware systems. Quite different IMHO.

**Comment by gseva** [comment]
@nodiscc 
I agree with the IoT subsection. It also could be outside of automation, along with projects like Freeboard and Phant.

On the other hand, i think that Node RED can be used easily outside the IoT environment and without hardware in mind, as it helps you creating logical flows where inputs can be APIs, feeds, sockets, files, etc.

File: README.md [hunk 2]
```diff
@@ -118,6 +118,7 @@ _Business Intelligence_
   * [Healthchecks](https://healthchecks.io/) - Django app which listens for pings and sends alerts when pings are late. ([Source Code](https://github.com/healthchecks/healthchecks)) `BSD` `Python`
   * [Huginn](https://github.com/cantino/huginn) - Allows you to build agents that monitor and act on your behalf - `MIT` `Ruby`
   * [Http2pic](https://http2pic.haschek.at/) - Website screenshots/renderer. It uses the wkhtmltox to render websites with various options. ([Source Code](https://github.com/chrisiaut/http2pic)) `Apache 2.0` `PHP/Javascript`
+  * [Node RED](http://nodered.org/) - A browser-based flow editor that helps you wiring hardware devices, APIs and online services to create IoT solutions. ([Source Code](https://github.com/node-red/node-red)) `Apache 2.0` `Nodejs`
```
Comment (AndyR207) [inline README.md]: Please change the language to `Apache`

**Comment by gseva** [comment]
@AndyR207 Done

**Comment by AndyR207** [comment]
Looks good to me.

**Pull request merged**